### PR TITLE
N&N for Theme Improvements

### PR DESCRIPTION
**Pull request by BeckerWdf**
Refers to
https://github.com/eclipse-platform/eclipse.platform.ui/issues/2114

### CODE REVIEW

File: news/4.33/platform.html [hunk 1]
```diff
@@ -141,6 +141,34 @@ <h2>Preferences </h2>
...
+        <p>
+        We have been working hard on an improved version of the light theme that follows these 
+        design decisions. These improvement are not yet finished but are available as a preview 
+        version. 
+        </p>
+        <p>
+        You can try it out by selecting <em>Light (Preview)</em> on the <b>General > 
+        Appearance</b> preference page. We are keen on your feedback. Please 
+        <a href="https://github.com/eclipse-platform/eclipse.platform.ui/issues/2114">give your 
+        feedback</a> and we will look into it.
```
Comment (sratz) [inline news/4.33/platform.html:167]: `See issue #2114 for more details and the motivation behind these changes and to give your feedback.`

File: news/4.33/platform.html [hunk 2]
```diff
@@ -141,6 +141,34 @@ <h2>Preferences </h2>
...
+        </p>
+        <p><img style="max-width: 600px;" src="images/preview-theme.png" alt="Preview Theme"/></p>
+      </td>
```
Comment (sratz) [inline news/4.33/platform.html:170]: Add a paragraph that we are planning to make the new theme the default in the next release?

**Comment by BeckerWdf** [comment]
Screenshot 2024-08-20 at 14 16 10

File: news/4.33/platform.html [hunk 3]
```diff
@@ -141,6 +141,38 @@ <h2>Preferences </h2>
...
+        </p>
+        <ol>
+          <li>Lightweight view tab design</li>
+          <li>Flat look - No use of 3D gradients</li>
+          <li>Views use a darker background color.</li>
+        </ol>
+        <p>
+        We have been working hard on an improved version of the light theme that follows these 
+        design decisions. These improvement are not yet finished but are available as a preview 
+        version.
```
Comment (HannesWell) [inline news/4.33/platform.html:161]:
```suggestion
        design decisions. These improvement are not yet completed but are available as a preview. 
```

File: news/4.33/platform.html [hunk 4]
```diff
@@ -141,6 +141,38 @@ <h2>Preferences </h2>
...
     </td>
   </tr>
+  <tr id="new-light-theme"> <!-- https://github.com/eclipse-platform/eclipse.platform.ui/issues/2114 -->
+      <td class="title">Preview Version of Improved Light Theme</td>
+      <td class="content">
+        <p>
+        The Eclipse default light theme has not been updated in years. It is not state 
+        of the art anymore. All major IDEs have shifted to a different light theme style.
+        When comparing the different IDEs it is easy to pick out the design decisions 
+        they all share:
```
Comment (HannesWell) [inline news/4.33/platform.html:151]: May this paragraph should be formulated more positively and should should mention less prominently that the redesign is ('just') inspired by other IDEs?
For example
```suggestion
        After many years of stability we have reworked the light theme of Eclipse to follow more modern design principles, similar to other major IDEs:
```

**Review by HannesWell — COMMENTED** [review]
Thanks for this (and in general for all the last minute adjustments!).
I have two suggestions below.

**Comment by BeckerWdf** [comment]
image

**Comment by BeckerWdf** [comment]
image

**Review by HannesWell — APPROVED** [review]
Looks good. Thank you!

**Comment by HannesWell** [comment]
@BeckerWdf before submission, can you please squash all commits into one?

**Pull request merged**